cmd/fabric-ca-client/command/config.go

```diff
@@ -343,7 +350,7 @@ func (c *ClientCmd) createDefaultConfigFile() error {
 
 	user := ""
 	var err error
-	if c.requiresUser() {
+	if c.requiresUser() && !c.iamEnroll {
```

Comment:
should add some comment as tag for changes in old files, for easy merge with upstream later

Reply:
ok.